bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -20,7 +23,7 @@ const takePresenterRole = () => makeCall('assignPresenter', Auth.userID);
 
 export default {
   isUserPresenter: () => Users.findOne({ userId: Auth.userID }).presenter,
-  isUserModerator: () => Users.findOne({ userId: Auth.userID }).moderator,
+  isUserModerator: () => Users.findOne({ userId: Auth.userID }).role === ROLE_MODERATOR,
```

Comment:
IMO would be better use the service [mapUser](https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/services/user/mapUser.js) that does that work already or add this field in addUser handler, something like  `user.moderator = user.moderator || false`